proto-public/pbdataplane/dataplane.proto

```diff
@@ -76,16 +78,27 @@ enum ServiceKind {
 }
 
 message GetEnvoyBootstrapParamsResponse {
-  ServiceKind service_kind = 1;
-  // service is be used to identify the service (as the local cluster name and
+  reserved 1;
```

Comment:
This field was unused by consul-dataplane and as it's not necessary in V2, it's safe to remove completely rather than deprecate.